Jenkinsfile

```diff
@@ -1 +1 @@
-elifeFormula('journal', '/srv/journal', null, ['s1804'])
+elifeFormula('journal', '/srv/journal', null, ['s1804', 'sNextSalt'])
```

Comment:
weird name, didn't know camel case was allowed?

Reply:
allowed by whom? I'm open to suggestions, really. I didn't know uppercase characters in bucket names were disallowed, so I'm fixing that for this. `snextsalt` is even weirder. `s2018-4-salt` is a little obscure and not much shorter than `standalone-next-salt`.

Reply:
resolved, we went with `snsalt`